src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -177,6 +181,14 @@ export const DepartureDetailsScreenComponent = ({
                 {title ?? t(DepartureDetailsTexts.header.notFound)}
               </ThemeText>
             </View>
+            {travelAidEnabled && (
```

Comment:
Just a small reminder here that this button is also affected by the toggle in the settings. But that toggle is in a separate PR.

Reply:
Yes, we need to fix that once your PR is merged 👍